<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassicMenuWidget</class>
 <widget class="QWidget" name="ClassicMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>100</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>18</number>
     </property>
     <item>
      <widget class="QPushButton" name="mpClients">
       <property name="text">
        <string>Клиенты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpApartment">
       <property name="text">
        <string>Квартиры</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpHome">
       <property name="text">
        <string>Дома/Участки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpRent">
       <property name="text">
        <string>Аренда</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpFindObject">
       <property name="text">
        <string>Поиск объекта</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>18</number>
     </property>
     <item>
      <widget class="QPushButton" name="mpMessages">
       <property name="text">
        <string>Сообщения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpAddressEditor">
       <property name="text">
        <string>Редактор адресов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpAdmin">
       <property name="text">
        <string>Администрирование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpSync">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Синхронизация БД</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpExit">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
